src/main/resources/org/jenkinsci/plugins/codesonar/CodeSonarPublisher/help-aid.html

```diff
@@ -1 +1,5 @@
-<div>Use this attribute to override analysis id</div>
\ No newline at end of file
+<div>
+    Use this attribute to override analysis id.

```

Comment:
You can't override analysis ID?

Reply:
We like the suggested text in your later comment here.  We will remove the use of "override" everywhere here.